nomad/blocked_evals_test.go

```diff
@@ -174,7 +175,10 @@ func TestBlockedEvals_UnblockEscaped(t *testing.T) {
 	}
 
 	blocked.Unblock("v1:123", 1000)
+	waitBrokerEnqueued(t, blocked, broker)
+}
 
+func waitBrokerEnqueued(t *testing.T, blocked *BlockedEvals, broker *EvalBroker) {
```

Comment:
Thank you so much for extracting this code!  It was quite repetitive.

Looking at the function, it seems to assert that the blocked evaluations made it to eval broker.  So maybe `requireBlockedEvalsEnqueued` is more appropriate.

Also, is `brokerStats.TotalReady` always expected to be 1?  Does it it depend on how many evaluations are blocked in the test?

Also, while you are here, I'd suggest updating the error message to be more meaningful than `bad: `